harness/nbjunit/src/org/netbeans/junit/ParametricTestSuite.java

```diff
@@ -36,23 +36,28 @@ public ParametricTestSuite() {
     
     /**
      * Constructs a ParametricTestSuite with the given name.
+     * @param name name of the suite
      */
     public ParametricTestSuite(String name){
         super(name);
     }
     
     /**
      * Returns an array of testcases for the given parameter.
+     * @param parameter parametre to retrieve the test case frome
+     * @return array of testcases
      */
     protected abstract ParametricTestCase[] cases(Object parameter);
     /**
      * Returns an array of parameters for this suite.
+     * @return array or parameter
```

Comment:
array **of** parameters

Reply:
@ebarboni wait, you did that wrong ;)

its always an array, and it contains parameters. It is not sometimes a parameter.

`or -> of`